docs/cloud/misc/sso.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "SSO"
+---
+
+###
+Marathon Cloud provides Single Sign-On (SSO) through SAML configuration.
+
+### Domain verification
+To set up SAML, you must verify ownership of your domain(s). To add your domain, visit the [Domains page](https://cloud.marathonlabs.io/domains) and follow the instructions to add it. After adding the domain, you'll need to add a TXT record to your DNS settings. DNS record updates typically take 5 to 10 minutes, but in some cases, it may take up to 72 hours.
```

Comment:
```suggestion
Setting up SAML requires verification of ownership of your domain(s). To add your domain, visit the [Domains page](https://cloud.marathonlabs.io/domains) and follow the instructions. After adding the domain, you'll need to add a TXT record to your DNS zone file. DNS record updates typically take 5 to 10 minutes, but in some cases, it may take up to 72 hours.
```